includes/Integrations/Elementor/Elementor.php

```
@@ -248,17 +322,146 @@ public function register_category( $elements_manager ) {
+
+    /**
+     * Flush rewrite rules when Elementor saves a page containing the User Directory widget
+     *
+     * @since 4.3.1
+     *
+     * @param int   $post_id The post ID.
+     * @param array $data    The Elementor data.
+     *
+     * @return void
+     */
+    public function maybe_flush_rules_on_elementor_save( $post_id, $data ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$data'. (undefined)

(UnusedFormalParameter)